zebedee-reader/src/main/java/com/github/onsdigital/zebedee/content/page/base/PageDescription.java

```diff
@@ -128,9 +128,8 @@ public void setMetaDescription(String metaDescription) {
         this.metaDescription = metaDescription;
     }
 
-
     public boolean isNationalStatistic() {
-        return nationalStatistic == null ? false : nationalStatistic;
+        return nationalStatistic != null && nationalStatistic;
```

Comment:
I think you've already done so but can you make sure you update all methods in this class of this type to follow the same pattern you've applied here. Any small consistency wins we can get through is plus for me.

Reply:
Thanks @daiLlew! I thought I did but just double checked and yes, all boolean getters in this class follow this pattern now 🙂